src/applications/personalization/dashboard/dashboard-entry.jsx

```diff
@@ -12,7 +12,7 @@ import manifest from './manifest.json';
 const url = manifest.rootUrl;
 
 startApp({
-  component: <DashboardWrapper version="1" rootUrl={url} />,
```

Comment:
`version` stopped being used a long time ago. And `rootUrl` is no longer used. Previously it was being passed to the `Dashboard` component that is removed in this PR